Ionic3.gitignore

```diff
@@ -0,0 +1,58 @@
+# Log and tmp data
+*~
+*.sw[mnpcod]
+*.log
+*.lock
+*.tmp
+*.tmp.*
+log.txt
+
+# NPM (Node Package Manager) files
+npm-debug.log*
+node_modules/
+tmp/
+temp/
+.package.tmp.json
+
+# Preprocessors cache files
+.idea/
```

Comment:
Many of these rules are actually IDE-specific rules, which we have defined under the `Global` subfolder. Please focus the template on what is important for using Ionic.